### PR TITLE
[PHP] sw-fw-less performance optimization

### PR DESCRIPTION
**Pull request by luoxiaojun1992**

### CODE REVIEW

**Comment by luoxiaojun1992** [comment]
1. Opened composer autoloader optimization switch
2. Removed unused provider configuration 
3. Add object pool configuration (to avoid creating new objects)

**Comment by luoxiaojun1992** [comment]
Could you please review the PR ?  @waghanza  Thanks.

File: php/sw-fw-less/composer.json [hunk 1]
```diff
@@ -21,5 +21,8 @@
       "App\\": "app/"
     }
   },
+  "config": {
+    "optimize-autoloader": true
```
Comment (waghanza) [inline php/sw-fw-less/composer.json:25]: not need, this will be done for all php frameworks (using the command line)

Comment (luoxiaojun1992) [reply php/sw-fw-less/composer.json]: Ok. It has been removed.

**Pull request merged**